config/projects-prod/ntap-cnf-cell-project

```diff
@@ -0,0 +1,47 @@
+stack_name: ntap-cnf-cell-project
+
+stack_tags:
+  Department: SCCE
+  Project: Neurofibromatosis
+  OwnerEmail: robert.allaway@sagebase.org
+  CostCenter: NTAP-JHU | 30110  # Valid values here: https://sagebionetworks.jira.com/l/c/0PyR5JSM
+
+parameters:
+
+  S3ReadWriteAccessArns:
+    # (REQUIRED) Step 3: Replace the email below with your '@sagebase.org' address
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/robert.allaway@sagebase.org
+
+    # (Optional) Step 4: Uncomment and update the following line(s) to grant additional users with read/write access
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jineta.banerjee@sagebase.org
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruno.grande@sagebase.org
+
+  # (Optional) Step 5: Uncomment and update the following line(s) to grant additional users with read-only access
+  # S3ReadOnlyAccessArns:
+  #   - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/thomas.yu@sagebase.org
+  #   - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jason.hwee@sagebase.org
+
+  # (Optional) Step 6: Uncomment and update the following lines to change the S3 bucket lifecycle configuration,
+  #                    which cannot be changed as long as 'AllowSynapseIndexing' is enabled (default)
+  # AllowSynapseIndexing: Disabled
+  # EnableDataLifeCycle: Enabled
+  # LifecycleDataTransition: '30'  # In days
+  # LifecycleDataStorageClass: INTELLIGENT_TIERING
+
+  # (Optional) Step 7: Uncomment and update the following line to change when objects are deleted automatically
+  LifecycleDataExpiration: '730'  # In days (i.e., 2 years)
```

Comment:
If you want to the data files to be deleted after 2 years, you'll need to disable Synapse indexing by uncommenting the lines under Step 6. 

Admittedly, the separation between Steps 6 and 7 suggests otherwise, so I pushed a clarification to the `main` branch. 

```suggestion
  # (Optional) Step 6: Uncomment and update the following lines to change the S3 bucket lifecycle configuration,
  #                    which cannot be changed as long as 'AllowSynapseIndexing' is enabled (default)
  AllowSynapseIndexing: Disabled
  EnableDataLifeCycle: Enabled
  LifecycleDataTransition: '30'  # In days
  LifecycleDataStorageClass: INTELLIGENT_TIERING

  # (Optional) Step 7: Uncomment and update the following line to change when objects are deleted automatically
  LifecycleDataExpiration: '730'  # In days (i.e., 2 years)
```